generic/tests/boot.py

```diff
@@ -39,14 +40,16 @@ def run(test, params, env):
             session.close()
     if params.get("reboot_method"):
         for vm in vms:
-            error_context.context("Reboot guest '%s'." % vm.name, test.log.info)
+            error_context.context("Reboot guest '%s'." % vm.name,
+                                  test.log.info)
             if params["reboot_method"] == "system_reset":
                 time.sleep(int(params.get("sleep_before_reset", 10)))
             # Reboot the VM
             if serial_login:
                 session = vm.wait_for_serial_login(timeout=timeout)
             else:
-                session = vm.wait_for_login(timeout=timeout)
+                session = vm.wait_for_login(nic_index=0,
```

Comment:
`nic_index=0` is the default value, but it's okay for me here. Also some code style fix.